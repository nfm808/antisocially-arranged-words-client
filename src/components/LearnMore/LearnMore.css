.LearnMore {
  background: #F2F2F2;
}