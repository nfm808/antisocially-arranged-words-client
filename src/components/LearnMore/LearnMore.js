import React from 'react';
import CahContext from '../../cahContext';
import { pickRandomCards } from '../../cahHelper';
import BlackCard from '../BlackCard/BlackCard';
import './LearnMore.css';


export default function LearnMore() {
  return (
    <section className="LearnMore">
      <h2 className="LearnMore--head">Play an online adaptation of Cards Against Humanity®️</h2>
      <CahContext.Consumer>
        { ({whiteCards, blackCards}) => 
          ( !whiteCards || !blackCards) ? null
          : <BlackCard card={pickRandomCards(blackCards, 1)} />

        }
      </CahContext.Consumer>
  </section>
)
}
