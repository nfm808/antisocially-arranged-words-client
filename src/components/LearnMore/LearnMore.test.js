import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LearnMore from './LearnMore';

describe('<LearnMore />', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<LearnMore />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})