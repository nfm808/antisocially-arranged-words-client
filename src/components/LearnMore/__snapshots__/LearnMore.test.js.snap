// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LearnMore /> renders without crashing 1`] = `
<section
  className="LearnMore"
>
  <h2
    className="LearnMore--head"
  >
    Play an online adaptation of Cards Against Humanity®️
  </h2>
  <ContextConsumer>
    <Component />
  </ContextConsumer>
</section>
`;
